GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -199,6 +199,11 @@ public override string GetNamedPipeName(string enlistmentRoot)
             return WindowsPlatform.GetNamedPipeNameImplementation(enlistmentRoot);
         }
 
+        public override string GetGVFSServiceNamedPipeName()
+        {
+            return GVFSConstants.Service.ServiceName + ".pipe";

```

Comment:
Question - On `POSIXPlatform.cs` we are dynamically getting the root directory path for `Service` and constructing the pipe name by appending `.pipe` string to it. But here on Windows, we are using a fixed `<service_name>.pipe` string. Why is this different between platforms - where does the pipe file get created on Windows.

Reply:
> where does the pipe file get created on Windows.

On Windows pipes are not files, but exist in a special namespace for Pipes: `\.\pipe\<NAME>`

Reply:
https://docs.microsoft.com/en-us/windows/desktop/ipc/pipe-names

Reply:
Since in *NIX pipes exist in the unified file system namespace they need to go somewhere under `/`.
A pipe with a simple name gets created directly under `Path.GetTempPath()` with `"CoreFXPipe_"` prefix.

See [here](https://github.com/dotnet/corefx/blob/e12bfba58e3581dc475e804688bef1c66d70226c/src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs#L49-L52) for more information.